src/romaine/parser/simple.py

```diff
@@ -0,0 +1,181 @@
+class SimpleParser(object):
+    """
+        Gherkin single line parser for Romaine core.
+    """
+    def is_white(self, line):
+        """
+            Takes a line and determines whether it is pure whitespace.
+
+            Keyword arguments:
+            line - Line to test.
+
+            Returns:
+            True if the line is purely whitespace. False otherwise.
+        """
+        line = line.strip()
```

Comment:
Yep the above proposal is good - it might be good to mention this difference in the docstring, but it should be reasonably obvious from the proposed implementation.